spec/javascripts/components/checkboxes-spec.js

```diff
@@ -3,12 +3,12 @@
 
 describe('Checkboxes component', function () {
   function loadCheckboxesComponent () {
-    var checkboxes = new GOVUK.Modules.Checkboxes()
+    var checkboxes = new GOVUK.Modules.GovukCheckboxes()
     checkboxes.start($('.gem-c-checkboxes'))
   }
 
   var FIXTURE =
-  '<div id="checkboxes-1ac8e5cf" class="gem-c-checkboxes govuk-form-group " data-module="checkboxes">' +
+  '<div id="checkboxes-1ac8e5cf" class="gem-c-checkboxes govuk-form-group " data-module="govuk-checkboxes">' +
```

Comment:
This means we won't support [conditional revealing](https://components.publishing.service.gov.uk/component-guide/checkboxes/checkbox_items_with_conditional_reveal), so we're fixing one example, but we're breaking another.